src/main/scripts/festerize/README.md

```diff
@@ -2,6 +2,10 @@
 
 Uploads CSV files to the Fester IIIF manifest service.
 
+After Fester processes the CSVs in order to create or update any IIIF collections or manifests, it updates and returns those CSV files to the user.
```

Comment:
If the CSV contains a work that has never been festerized before, then a column is added to the CSV that provides the IIIF Manifest URL. If the CSV contains pages, the values in the CSV are used to expand or revise a previously festerized work record.

Reply:
That's my understanding of what happens. I know you have that in the next paragraph to some extent so you might want to revise to merge my comment with your description.